### PR TITLE
chore: pin examples to 1.14 minor

### PR DESCRIPTION
**Pull request by mcansh**
don't want these to block our release of v2, but don't want them all busted either or have deprecation warnings everywhere. we can updated them to v2 incrementally.

### CODE REVIEW

File: __template/package.json [hunk 1]
```diff
@@ -8,16 +8,16 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@remix-run/node": "*",
-    "@remix-run/react": "*",
-    "@remix-run/serve": "*",
+    "@remix-run/node": "~1.14.2",
```
Comment (MichaelDeBoey) [inline __template/package.json:11]: What about using carets (`^`) instead, so we can still have v1.15 if we would release that?

```suggestion
    "@remix-run/node": "^1.14.2",
```

Comment (mcansh) [reply __template/package.json:11]: 1.15 will introduce the deprecations which is what we're trying to avoid here

Comment (MichaelDeBoey) [reply __template/package.json:11]: Just the deprecation warnings, no?
So everything would still work, no?

Or is that the thing we don't want to have in the examples: the deprecation warnings?

**Review by MichaelDeBoey — APPROVED** [review]

File: __template/package.json (hunk 1 above)
Comment (mcansh) [reply __template/package.json:11]: > Just the deprecation warnings, no?
> So everything would still work, no?

correct, everything will still work, but with deprecation warnings - we have a plan to update/ask contributors of examples to update, but that's not needed actively

> should we also update the stacks?
stacks can be updated around the time of 1.15 being released to opt into future flags

Comment (MichaelDeBoey) [reply __template/package.json:11]: Gotcha! 👍

**Review by machour — APPROVED** [review]

**Pull request merged**